apps/happy-blocks/bin/build-translations-manifest.js

```diff
@@ -3,37 +3,50 @@
 /* eslint-disable import/no-nodejs-modules */
 const fs = require( 'fs' );
 const path = require( 'path' );
+const glob = require( 'glob' );
 
 const PROJECT_DIR = path.resolve( __dirname, '..' );
 const ROOT_DIR = path.resolve( PROJECT_DIR, '..', '..' );
-const CHUNKS_MAP_PATH = path.resolve( PROJECT_DIR, 'dist', 'chunks-map.json' );
+const CHUNKS_MAP_PATHS = glob.sync(
+	path.resolve( PROJECT_DIR, 'block-library/*/dist/', 'chunks-map.json' )
```

Comment:
It seems like the `chunks-map.json` files are generated in `'block-library/*/build/'` instead of `dist`.

Reply:
Nice catch! I will update that now.